style.css

```diff
@@ -0,0 +1,482 @@
+}
+
+.about {
+  margin-top: 100px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  margin-left: 10px;
+}
+
+.about_datile {
+  font-family: 'Poppins', sans-serif;
+  width: 400px;
+  margin-left: 10px;
+  font-weight: 400;
+  font-size: 16px;
+}
+
+.contact_list1 ul {
+  margin-left: 10px;
+}
+
+.contact_list1 {
+  margin: 0 10px;
+  list-style: none;
+  display: inline-block;
+}
+
+.list {
+  display: inline-block;
+  margin: 0 10px;
+}
+
+.contact_list1 h3 {
+  margin: 10px 10px;
+  font-family: 'Nunito Sans', sans-serif;
+  font-weight: 500;
+  color: #6070ff;
+}
+
+.resume {
+  background: transparent;
+  padding: 16px;
+  margin-top: 40px;
+  color: #6070ff;
+  border: 1px solid #6070ff;
+  border-radius: 16px;
+  font-weight: 500;
+  font-size: 25px;
+  cursor: pointer;
+  outline: none;
+}
+
+.languages {
+  margin: 40px 10px 0 10px;
+  font-family: 'Nunito Sans', sans-serif;
+  font-size: 23px;
+  font-weight: 500;
+}
+
+.skills {
+  margin-top: 20px;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.add_skill {
+  width: 100%;
+  height: auto;
+  padding: 16px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  display: flex;
+  align-items: center;
+  margin: 10px 0;
+}
+
+.add_skill h3 {
+  font-size: 20px;
+  margin-left: 20px;
+  font-weight: 400;
+}
+
+.works {
+  margin-top: 40px;
+  width: 100%;
+  height: auto;
+}
+
+.work_skill {
+  background: transparent;
+  flex-direction: row;
+  border-bottom: 1px solid #dfe1e6;
+  border-radius: 0;
+  width: 100%;
+  height: auto;
+  padding: 16px;
+  display: flex;
+  align-items: center;
+  margin: 10px 0;
+}
+
+.work_skill h3 {
+  font-size: 19px;
+  font-weight: 400;
+}
+
+.work_skill img {
+  position: absolute;
+  right: 0;
+  width: 30px;
+  margin-right: 50px;
+}
+
+.full_form {
+  background: #6070ff;
+  background-image: url('Images/bottom_im.png');
+  background-repeat: no-repeat;
+  background-position-x: right;
+  width: 100%;
+  height: 700px;
+  padding: 16px;
+  position: absolute;
+  margin-top: -100px;
+  border-top-left-radius: 100px;
+  display: flex;
+  flex-direction: column;
+  justify-items: center;
+}
+
+.contact_me {
+  margin-top: 70px;
+  width: 100%;
+  height: auto;
+  text-align: center;
+  margin-left: 10px;
+}
+
+.contact_me h1 {
+  font-family: 'Nunito Sans', sans-serif;
+  font-size: 35px;
+  color: #fff;
+  font-weight: 700;
+}
+
+.contact_me p {
+  font-family: 'Poppins', sans-serif;
+  font-size: 15px;
+  color: #fff;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  margin-top: 30px;
+}
+
+#input {
+  width: 100%;
+  padding: 10px 12px;
+  outline: none;
+  border: none;
+  background: #fff;
+  margin: 10px 0;
+  border-radius: 5px;
+  font-family: 'Poppins', sans-serif;
+}
+
+#textpanel {
+  width: 100%;
+  padding: 10px 12px;
+  outline: none;
+  border: none;
+  background: #fff;
+  margin: 10px 0;
+  border-radius: 5px;
+  font-family: 'Poppins', sans-serif;
+}
+
+.submit {
```

Comment:
- [ ] I kindly suggest that you add the different button interactions including hover, active (when it's pressed), disabled, and enable (default state) as shown in the Figma template below:

![Screenshot from 2022-04-27 19-12-50](https://user-images.githubusercontent.com/52934128/165592912-7d8bf224-8e6b-467d-bb19-cf63cd2b40fd.png)

the button on the pressed state has the `background: #2230D2;` and on hover has the `background: #6070FF;`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #2230D2;
}
```
**for hover state**
```
button:hover {
    background-color: #6070FF;
}
```